{"name": "test_login[resume_save]", "status": "broken", "statusDetails": {"message": "TypeError: myReplace() missing 1 required positional argument: 'name'", "trace": "self = <test_01.Test_choice object at 0x00000253F5FDDD08>, name = 'resume_save'\n\n    @allure.feature('测试用例1')\n    @pytest.mark.parametrize('name', (case.caae_name[\"choice\"][\"login\"]))\n    def test_login(self,name):\n>       self.pytestObject(self,name)\n\ntest_01.py:13: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\base\\PytestObject.py:13: in pytestObject\n    assertion=case.run(module,name)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <base.CaseOperate.Case object at 0x00000253F5F5A388>, class_name = 'choice', name = 'resume_save'\n\n    def run(self, class_name, name):\n        request_list = self.request[class_name][name]\n        url = request_list[\"url\"]\n>       body = self.myReplace(json.dumps(request_list['body']))\nE       TypeError: myReplace() missing 1 required positional argument: 'name'\n\n..\\base\\CaseOperate.py:60: TypeError"}, "parameters": [{"name": "name", "value": "'resume_save'"}], "start": 1605251118019, "stop": 1605251118019, "uuid": "16b56975-fdcd-46cd-a232-5fc57501909a", "historyId": "14e3421c166be4fa410f05426eebc418", "testCaseId": "29833af1bc62bae296571ec4b1662ac8", "fullName": "test_01.Test_choice#test_login", "labels": [{"name": "feature", "value": "测试用例1"}, {"name": "suite", "value": "test_01"}, {"name": "subSuite", "value": "Test_choice"}, {"name": "host", "value": "LAPTOP-4V3QBUP4"}, {"name": "thread", "value": "4880-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_01"}]}